docker/Dockerfile_stack

```diff
@@ -45,8 +44,10 @@ RUN apt-get update && apt-get install -y \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-RUN pip install --upgrade --no-cache-dir pip && \
-    pip install --no-cache-dir sphinx==1.4 sphinx_rtd_theme breathe==4.6
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+    python get-pip.py && \
+    pip install --upgrade --no-cache-dir pip && \
+    pip install --no-cache-dir sphinx==1.4 sphinx_rtd_theme breathe==4.7
```

Comment:
Now I realize that upgrading breathe to 4.7 is not necessarily ok.  I would have to update the base image for the tests that run on Travis as well.